site/styles/_awards.scss

```diff
@@ -27,7 +27,8 @@
 
 .meta-awards-icon-overlay {
   @extend .text-center;
-
+  @extend .d-flex;
```

Comment:
Can you please replace these bootstrap classes with the actual CSS.

Reply:
- replaced these three